packages/database/test/exp/integration.test.ts

```diff
@@ -138,6 +141,43 @@ describe('Database@exp Tests', () => {
     unsubscribe();
   });
 
+  it('can connected to emulator', async () => {
```

Comment:
Nit: some typos. "can connected to" here and in 160, "chnage" in 151

Reply:
Fixed!